content/api/events/catalog-of-events.md

```diff
@@ -184,9 +184,9 @@ single test.
 #### To turn off all uncaught exception handling
 
 ```javascript
-// likely want to do this in a support file
+// you'll like want to do this in a supportFile
```

Comment:
Can you move these comments into plain text above each code block and link `supportFile` to that URL?